enterprise/server/tasksize_model/tasksize_model.go

```diff
@@ -0,0 +1,245 @@
+package tasksize_model
```

Comment:
This is *really* cool, but I'd much prefer to just use a straight up tensorflow lib for doing this rather than our own so that if we make this model better later we are well supported -- can we try that?

Reply:
Done - switched to a TF model. Each prediction takes around 10 usec. I will check in the tooling for generating the TF repo (currently hosted at https://github.com/bduffany/go-tensorflow) in a separate PR